airflow/www/templates/airflow/dag_details.html

```diff
@@ -29,7 +29,7 @@
       <a
           class="btn"
           style="border: none; background-color:{{ State.color(state)}}; color: {{ State.color_fg(state) }};"
-          href="/admin/taskinstance/?flt0_dag_id_equals={{ dag.dag_id }}&flt2_state_equals={{ state }}">
+          href="{{ url_for('admin.index')}}taskinstance/?flt0_dag_id_equals={{ dag.dag_id }}&flt2_state_equals={{ state }}">
```

Comment:
This is much difficult than I though:
- ```url_for('admin.taskinstance')}}``` does not work (template render exception)
- ```url_for('airflow.task_instances')}}``` works but the URL pointed is wrong (wants to go to ```/<urlprefix>/admin/airflow/object/taskinstance```)